scripts/local_testnet/validator_client.sh

```diff
@@ -28,6 +28,7 @@ exec lighthouse \
 	$BUILDER_PROPOSALS \
 	--datadir ${@:$OPTIND:1} \
 	--testnet-dir $TESTNET_DIR \
+	--builder-proposals \
```

Comment:
Note: there is a `-p` option to enbable this flag as well in `start_local_testnet.sh`.